### PR TITLE
Fixed display of cached hit ratio

### PR DESCRIPTION
**Pull request by albertlast**
The array didn't exists any more for the next echo whre the implode use this var,
so i define this on the start of the if.

### CODE REVIEW

File: Sources/Logging.php [hunk 1]
```diff
@@ -260,6 +260,7 @@ function displayDebug()
 
 	if (!empty($modSettings['cache_enable']) && !empty($cache_hits))
 	{
+		$missed_entires = array();
```
Comment (NanoSector) [inline Sources/Logging.php]: typo: missed_entries

Comment (NanoSector) [reply Sources/Logging.php]: it's actually broken in the original code too. might be best to just fix it to avoid confusion.

Comment (albertlast) [reply Sources/Logging.php]: already done

**Review by colinschoen — APPROVED** [review]
LGTM approval

**Pull request merged**